### ISSUE TITLE
How to train through a stream?  Or a way that alleviates large in-memory requirement for `model.train()` usage?

### ISSUE THREAD

**Issue by niftylettuce** [mljs/naiveBayes#13]
If you have a large dataset, of perhaps 5 GB+, then it is not efficient to do an in-memory `model.train(sets, labels)` approach whereas `sets` and `labels` could consist of hundreds of thousands of Arrays.  What is an alternative approach to stream the sets and labels to training?
